roles/postgresql/molecule/default/verify.yml

```diff
@@ -0,0 +1,24 @@
+---
+# This is an example playbook to execute Ansible tests.
```

Comment:
Update stock comment.

Anything to understand here about the setup that made this work? The verify test looks good

Reply:
It doesn't work - did it work for you? I wrote in the PR description that I was still messing with it - it isn't using Francis's recommended approach of installing postgres locally yet.

Reply:
Oh, I misread the molecule output. No, it fails with an error when it's trying to pull the docker image.

Reply:
Yeah, you'd need to build it locally first with `docker build -t pulpostgres .` in the directory containing the `Dockerfile`. That image doesn't exist on docker hub to pull down.

Reply:
Moved testing to #52.